ApiBundle/Controller/ControllerTrait/HandleRequestDataTable.php

```diff
@@ -5,6 +5,7 @@
 use Doctrine\ODM\MongoDB\DocumentRepository;
 use OpenOrchestra\BaseApi\Transformer\TransformerInterface;
 use Symfony\Component\HttpFoundation\Request;
+use OpenOrchestra\BaseApi\Facade\FacadeInterface;
```

Comment:
As this is a trait, I don't think that the use are useFull, especially for hte comment